templates/data/postgresql.html

```diff
@@ -1,481 +1,561 @@
+          <div class="row">
+            <div class="col-4 col-medium-2 col-small-3">
+              <h3 class="u-align-text--left u-no-margin--bottom">Self-managed</h3>
+            </div>
+            <div class="col-2 col-medium-1 col-small-1">
+              <h3 class="u-align-text--right u-no-margin--bottom">From $500</h3>
+              <p class="u-align-text--right u-text--muted">Per node, per year</p>
+            </div>
+          </div>
+          <p>
+            24/7, weekday or advanced fire-fighting support for PostgreSQL from Canonical for up to 10 years per release track.
+          </p>
+          <hr class="is-muted" />
+          <div class="p-section--shallow u-no-margin--bottom">
+            <ul class="p-list--divided u-no-margin--bottom">
+              <li class="p-list__item is-ticked">Expert troubleshooting on hand</li>
+              <li class="p-list__item is-ticked">Security and bug fixes</li>
+              <li class="p-list__item is-ticked">Full stack coverage</li>
+            </ul>
+          </div>
         </div>
       </div>
-      <hr class="p-rule"/>
-      <div class="row">
-        <div class="col-3 col-medium-2">
-          <h3 class="p-heading--5">Applications managing geospatial data </h3>
-        </div>
-        <div class="col-3 col-medium-2">
-          <p>The <a href="https://postgis.net/">PostGis</a> extension of PostgreSQL is an excellent choice to support use cases managing spatial data.</p>
-        </div>
-        <div class="col-3 col-medium-2">
-          {{ 
-            image (
-            url="https://assets.ubuntu.com/v1/acaae572-Applications managing geospatial data@3x.png",
-            alt="",
-            width="901",
-            height="600",
-            hi_def=True,
-            loading="lazy"
-            ) | safe
-          }}
-        </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50 p-section--shallow">
+      <hr />
+      <div class="col">
+        <h2>PostgreSQL use cases</h2>
       </div>
-      <hr class="p-rule"/>
-      <div class="row">
-        <div class="col-3 col-medium-2">
-          <h3 class="p-heading--5">FinOps and banking</h3>
-        </div>
-        <div class="col-3 col-medium-2">
-          <p>PostgreSQL provides the transactional <a href="https://en.wikipedia.org/wiki/ACID">ACID</a> guarantees and the mature replication capabilities that finance systems need. The <a href="https://www.pgaudit.org/">pgAudit</a> extension can also complement PostgreSQL with the auditing capabilities that regulated industries require.</p>
+      <div class="col">
+        <p>
+          PostgreSQL is the world's most acclaimed open source database. Its popularity is rooted in its sound architectural foundations that make it a good fit for several use cases:
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-large-4 col-9 col-medium-6">
+        <hr class="is-muted" />
```

Comment:
```suggestion
        <hr class="is-muted p-rule" />
```
There is too much top padding for each list item, can you update all the `hr` to make it consistent?

Reply:
But isn't p-rule has been depreciated?

Reply:
Thanks for pointing that out! There is a new class `p-rule--muted` which should takeover the stylings of this `hr`. I believe this one isn't deprecated ([docs](https://vanillaframework.io/docs/patterns/rule#muted))

Reply:
I am not sure about this since the only class I have been using for this is `is-muted` for couple of PRs